calabash/Classes/Utils/LPDevice.m

```diff
@@ -512,6 +515,15 @@ - (BOOL) isLetterBox {
   }
 }
 
+- (BOOL) isIPhone10LetterBox {
+  if (![self isIPhone10Like]) { return NO; }
+  NSDictionary *dimensions = [self screenDimensions];
+  NSInteger screenModeHeight = [dimensions[@"height"] integerValue];
+  NSInteger screenBoundsHeight = [dimensions[@"bounds_portrait_height"] integerValue];
+
+  return screenModeHeight == 2001 && screenBoundsHeight == 667;
```

Comment:
Any case where `screenModeHeight > screenBoundsHeight` is per definition a letterbox case, right?

Reply:
Maybe?

I think it is possible that some apps will be scaled and not be in letter box.